app/web_conn.go

```diff
@@ -401,10 +392,12 @@ func (wc *WebConn) ShouldSendEvent(msg *model.WebSocketEvent) bool {
 	return true
 }
 
+// IsMemberOfTeam returns whether the user of the WebConn
+// is a member of the given teamId or not.
 func (wc *WebConn) IsMemberOfTeam(teamId string) bool {
 	currentSession := wc.GetSession()
 
-	if currentSession == nil || len(currentSession.Token) == 0 {
+	if currentSession == nil || currentSession.Token == "" {
 		session, err := wc.App.GetSession(wc.GetSessionToken())
 		if err != nil {
 			mlog.Error("Invalid session.", mlog.Err(err))
```

Comment:
This is just me trying to understand. Why do we need this code here but not in `ShouldSendEvent`? When we call `IsMemberOfTeam` it's possible we have already called `wc.GetSession()` without any such check.

Reply:
I think it's only there in case this method gets called from elsewhere too, because it was exported. But currently, this will always get called after `if !wc.IsAuthenticated() {` which always ensures that a session is set. So yes, it's not necessary, and maybe we can remove it once we unexport it. But we need to ensure that it does not get called with a nil session.